pkg/models/model_plan_and_operational_solution_raw.go

```diff
@@ -0,0 +1,39 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// ModelPlanAndOperationalSolutionRaw represents the combined data from the SQL query with unique field names
+type ModelPlanAndOperationalSolutionRaw struct {
```

Comment:
This structure exists to disambiguate fields with shared names between model plan and operational solution, such as `id` or `created_by`.

Reply:
On the one hand, I agree 100% with your concerns about redundancy in field definitions. I spent about half a day looking for a solution to this before falling back on the current implementation. Multiple tag definitions per your suggestion is an interesting idea, looking into this to see if it works.

Conversely, since we explicitly define our field selections in SQL rather than using wildcards it could be argued that the additional code brittleness is a mild extension of an existing architectural issue. That said, I’m wondering if we should consider wildcard usage and fall back on unit tests to assert deserialization success.